Lib/ssl.py

```diff
@@ -777,6 +777,10 @@ def version(self):
         current SSL channel. """
         return self._sslobj.version()
 
+    if HAS_TLSv1_3:
+        def verify_client_post_handshake(self):
+            return self._sslobj.verify_client_post_handshake()
```

Comment:
Do you want to always expose the method but raise `NotImplementedError` if no TLS 1.3 is available?